lib/src/game/base_game.dart

```diff
@@ -47,6 +43,12 @@ class BaseGame extends Game with FPSCounter {
         'Tapable Components can only be added to a BaseGame with HasTapableComponents',
       );
     }
+    if (c is Draggable) {
```

Comment:
I think that we could add a test for this, and for the others as well, what do you think?

Reply:
🧠 🧙‍♂️